RevenueCatUI/UIKit/PaywallFooterViewController.swift

```diff
@@ -36,15 +36,20 @@ public final class PaywallFooterViewController: PaywallViewController {
     /// `Offerings.current` will be used by default.
     @objc
     public init(offering: Offering? = nil) {
-        super.init(offering: offering, displayCloseButton: false)
+        super.init(offering: offering,
+                   fonts: DefaultPaywallFontProvider(),
+                   displayCloseButton: false)
     }
 
     @available(*, unavailable)
     override init(
         offering: Offering? = nil,
+        fonts: PaywallFontProvider,
```

Comment:
IMO It's weird to have a parameter without a default value after one with one... should it be the first parameter? Or should we give it a default value `DefaultPaywallFontProvider`?

Reply:
I know it's unusual in other languages, but argument labels are required in swift. If you look at the API tester, the different combinations all make sense.

I ordered it this way to be consistent with the `PaywallView` argument order. It also can't have a default value here because calling a constructor with no arguments would be ambiguous.